ecmascript/transforms/proposal/src/decorators/legacy.rs

```diff
@@ -773,12 +773,30 @@ impl Legacy {
                 });
             }
 
-            constructor
+            let decorate_stmts_insert_position = constructor
                 .body
                 .as_mut()
                 .unwrap()
                 .stmts
-                .extend(constructor_stmts)
+                .iter()
+                .position(move |stmt| {
```

Comment:
Is this `move` necessary?
I think closure does not use any variables from outer scopes.